site/utils/formUtils.ts

```diff
@@ -72,3 +72,23 @@ export const getDataInPages = <T>(pageNumber: number, data: T[]): T[] => {
     const endPoint = pageNumber * 10;
     return data.slice(startPoint, endPoint);
 };
+
+export const getStopType = (stopType: string | undefined) => {
+    let stop = "";
```

Comment:
you can remove this and just do return "Bus stop" on like 80 same for the others and then remove line 93. The breaks can then be removed also

Reply:
done changed to if statements